openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_capture.json

```diff
@@ -348,6 +348,31 @@
                             "type": "label",
                             "label": "<b>Show</b>"
                         },
+                        {
+                            "type": "boolean",
+                            "key": "xray",
+                            "label": "Toggle X-Ray Mode"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "udm",
+                            "label": "Use Default Material"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "wos",
+                            "label": "Wireframe On Shaded"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "jointXray",
+                            "label": "Toggle Joint X-Ray Mode"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "backfaceCulling",
+                            "label": "Enable Back Face Culling"
+                        },
```

Comment:
Could we move this to above the label **Show** and then add a label just above it that says **Shading**?

That way it matches the Maya Viewport's menu name for Shading.
Then I'd actually labelize the settings the same way as Maya does too:
![afbeelding](https://user-images.githubusercontent.com/2439881/192211240-854b9543-5243-4933-95d5-59abd33feff3.png)

So that would be like this refactor, but then moved higher.
```suggestion
                        {
                            "type": "label",
                            "label": "<b>Shading</b>"
                        }, 
                        {
                            "type": "boolean",
                            "key": "udm",
                            "label": "Use Default Material"
                        },
                        {
                            "type": "boolean",
                            "key": "wos",
                            "label": "Wireframe On Shaded"
                        },
                        {
                            "type": "boolean",
                            "key": "xray",
                            "label": "X-Ray"
                        },
                        {
                            "type": "boolean",
                            "key": "jointXray",
                            "label": "X-Ray Joints"
                        },
                        {
                            "type": "boolean",
                            "key": "backfaceCulling",
                            "label": "Backface Culling"
                        },
```

Reply:
You got it!